parcel_test.go

```diff
@@ -30,58 +31,79 @@ func getTestParcel() Parcel {
 
 // TestAddGetDelete проверяет добавление, получение и удаление посылки
 func TestAddGetDelete(t *testing.T) {
-	// prepare
-	db, err := // настройте подключение к БД
+	db, err := sql.Open("sqlite", "./tracker.db")
+	defer db.Close()
+	require.NoError(t, err)
 	store := NewParcelStore(db)
 	parcel := getTestParcel()
 
-	// add
-	// добавьте новую посылку в БД, убедитесь в отсутствии ошибки и наличии идентификатора
+	parcelNumber, err := store.Add(parcel)
+	assert.NoError(t, err)
+	assert.NotEmpty(t, parcelNumber)
 
-	// get
-	// получите только что добавленную посылку, убедитесь в отсутствии ошибки
-	// проверьте, что значения всех полей в полученном объекте совпадают со значениями полей в переменной parcel
+	parcelGet, err := store.Get(parcelNumber)
+	assert.NoError(t, err)
+	assert.Equal(t, parcel.Client, parcelGet.Client)
+	assert.Equal(t, parcel.Status, parcelGet.Status)
+	assert.Equal(t, parcel.Address, parcelGet.Address)
+	assert.Equal(t, parcel.CreatedAt, parcelGet.CreatedAt)
+
+	err = store.Delete(parcelNumber)
+	assert.NoError(t, err)
+	_, err = store.Get(parcelNumber)
+	assert.Equal(t, sql.ErrNoRows, err)
```

Comment:
Для сравнивания ошибок нужно использовать специализированную функцию
```suggestion
	require.ErrorIs(t, err, sql.ErrNoRows)
```

Reply:
Fixed